module/src/main/java/org/mule/extension/spring/internal/factory/ConstantFactoryBean.java

```diff
@@ -1,55 +0,0 @@
-/*
- * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
- * The software in this package is published under the terms of the CPAL v1.0
- * license, a copy of which has been included with this distribution in the
- * LICENSE.txt file.
```

Comment:
and here

Reply:
Hi @sabecasismulesoft  these files were removed in the original PR for JAVA 17 https://github.com/mulesoft/mule-spring-module/pull/205/files?file-filters%5B%5D=.java&show-deleted-files=true&show-viewed-files=false
so my guess si yes.